extension.js

```diff
@@ -179,11 +184,17 @@ function getMonitorArray() {
 }
 
 function moveWindow(direction) {
+	_log("---");
```

Comment:
This extra line makes it easier to distinguish log line groups for separate keystrokes.